vmdb/spec/models/ems_refresh/refreshers/vc_refresher_errors_spec.rb

```
@@ -2,6 +2,8 @@
 
 describe EmsRefresh::Refreshers::VcRefresher do
   before(:each) do
+    EmsRefresh.debug_failures = false
+
```

Same question on this one...can we just let it blow up?